src/chrome/content/rules/0xbadc0de.be.xml

```diff
@@ -29,20 +29,19 @@
 -->
 <ruleset name="0xbadc0de.be">
 
-	<target host="0xbadc0de.be" />
-	<target host="*.0xbadc0de.be" />
+	<target host=      "0xbadc0de.be" />
+	<target host= "blog.0xbadc0de.be" />
+	<target host="stats.0xbadc0de.be" />
+	<target host=  "www.0xbadc0de.be" />
 
-		<test url="http://blog.0xbadc0de.be/" />
-		<test url="http://stats.0xbadc0de.be/" />
-		<test url="http://www.0xbadc0de.be/" />
+	<test url=    "http://0xbadc0de.be/?p=67" />
+	<test url="http://www.0xbadc0de.be/?p=67" />
 
-
-	<!--	Redirect drops path but not args:
-							-->
-	<rule from="^http://(?:www\.)?0xbadc0de\.be/[^?]*"
+	<!--	Redirect drops path but not args:         -->
```

Comment:
You should also have a test URL that includes a path as well as args, ideally one that is linked to from somewhere on the site.